docs/jailer.md

```diff
@@ -280,10 +280,12 @@ Note: default value for `<api-sock>` is `/run/firecracker.socket`.
 ### Known limitations
 
 - When passing the --daemonize option to Firecracker without the --new-ns-pid
-  option, the Firecracker process will have a different pid than the Jailer
-  process. The suggested workaround to get Firecracker process's pid in this
-  case is using `--new-pid-ns` flag and read Firecracker's pid from the
-  `firecracker.pid` file present in the jailer's root directory.
+  option, the Firecracker process will have a different PID than the Jailer
```

Comment:
Isn't this always true when using the daemonize option, regardless of the `--new-pid-ns`?

Reply:
rephrased the comments to make the issue more clear.

Reply:
+1 to remove "without --new-ns-pid option".